DigitalLearningSolutions.Web/ViewModels/TrackingSystem/CourseSetup/CourseSetupViewModel.cs

```diff
@@ -3,14 +3,47 @@
     using System.Collections.Generic;
     using System.Linq;
     using DigitalLearningSolutions.Data.Models.Courses;
+    using DigitalLearningSolutions.Web.Helpers;
+    using DigitalLearningSolutions.Web.ViewModels.Common.SearchablePage;
 
-    public class CourseSetupViewModel
+    public class CourseSetupViewModel : BaseSearchablePageViewModel
     {
-        public CourseSetupViewModel(IEnumerable<CourseStatistics> courses)
+        public CourseSetupViewModel(
+            IEnumerable<CourseStatistics> courses,
+            IEnumerable<string> categories,
+            IEnumerable<string> topics,
+            string? searchString,
+            string sortBy,
+            string sortDirection,
+            string? filterBy,
+            int page
+        ) : base(searchString, page, true, sortBy, sortDirection, filterBy)
         {
-            Courses = courses.Select(course => new SearchableCourseStatisticsViewModel(course));
+            courses = courses.ToList();

```

Comment:
Is there any reason of doing IEnumerable<CourseStatistics> type courses as a param, then override it with .ToList(), then change it into Queryable, then never use it again?